src/main/java/slimeknights/tconstruct/smeltery/tileentity/TileSearedFurnace.java

```diff
@@ -143,6 +148,20 @@ protected boolean onItemFinishedHeating(ItemStack stack, int slot) {
   protected int getUpdatedInventorySize(int width, int height, int depth) {
     return 9 + (3 * width * height * depth);
   }
+  
+  protected void interactWithEntitiesInside() {
+      AxisAlignedBB bb = info.getBoundingBox().contract(1).offset(0, 0.5, 0).expand(0, 0.5, 0);
+
+      List<Entity> entities = getWorld().getEntitiesWithinAABB(Entity.class, bb);
+
+      for(Entity entity : entities) {
+          if (entity instanceof EntityMob) {
+              EntityLivingBase entityLiving = (EntityLivingBase) entity;
+
+              entityLiving.setHealth(0F);
```

Comment:
I think dealing fire or lava damage consistently is better than a simple `setHealth`.